graphene_django/tests/test_converter.py

```diff
@@ -166,14 +164,14 @@ def test_field_with_choices_convert_enum():
         help_text="Language", choices=(("es", "Spanish"), ("en", "English"))
     )
 
-    class TranslatedModel(models.Model):
+    class ChoicesModel(models.Model):
```

Comment:
This fixes a Django warning about the `TranslatedModel` being already registered:

> RuntimeWarning: Model 'test.translatedmodel' was already registered. Reloading models is not advised as it can lead to inconsistencies, most notably with related models.